### PR TITLE
fix: Tighten invitation proposal patterns

### PR DESCRIPTION
**Pull request by erights**
Staged on https://github.com/Agoric/agoric-sdk/pull/5422 "multiples"

Use tight proposal patterns on production invitations to, among other things, see if the Zoe prohibition on wash invitations is something we should bother preserving. (My assumption is not, but worth doing some of this experiment first.)

### CODE REVIEW

**Review by Chris-Hibbert — APPROVED** [review]

**Comment by erights** [comment]
Reviewers, especially for the occurrences of `EmptyProposal`, let me know if each one makes sense, and where the corresponding `offer` call happens. I was not able to find many of those `offer` calls, and so could not easily tell what proposal pattern is appropriate.

File: packages/governance/src/committee.js [hunk 1]
```diff
@@ -92,7 +93,12 @@ const start = (zcf, privateArgs) => {
                 return E(voteCap).submitVote(voterHandle, positions, 1n);
               };
 
-              return zcf.makeInvitation(continuingVoteHandler, 'vote');
+              return zcf.makeInvitation(
+                continuingVoteHandler,
```
Comment (Chris-Hibbert) [inline packages/governance/src/committee.js:101]: `continuingVoteHandler` is exercised by the Wallet. If it's ever not an `EmptyProposal`, I think that's under the user's control, but I have no idea how.

File: packages/governance/src/committee.js [hunk 2]
```diff
@@ -103,7 +109,12 @@ const start = (zcf, privateArgs) => {
     // This will produce unique descriptions because
     // makeCommitteeVoterInvitation() is only called within the following loop,
     // which is only called once per Electorate.
-    return zcf.makeInvitation(offerHandler, `Voter${index}`);
+    return zcf.makeInvitation(
+      offerHandler,
+      `Voter${index}`,
+      undefined,
+      EmptyProposal,
```
Comment (Chris-Hibbert) [inline packages/governance/src/committee.js:120]: yes, always empty.

[Example](https://github.com/Agoric/agoric-sdk/blob/ffe5e7ff3dd7ec1ffbbcd86e244ef167ed70c53b/packages/inter-protocol/test/swingsetTests/governance/vat-voter.js#L16)

File: packages/governance/src/electorateTools.js [hunk 3]
```diff
@@ -69,7 +70,12 @@ const getQuestion = (questionHandleP, questionStore) =>
  */
 const getPoserInvitation = (zcf, addQuestion) => {
   const questionPoserHandler = () => Far(`questionPoser`, { addQuestion });
-  return zcf.makeInvitation(questionPoserHandler, `questionPoser`);
+  return zcf.makeInvitation(
+    questionPoserHandler,
+    `questionPoser`,
+    undefined,
+    EmptyProposal,
```
Comment (Chris-Hibbert) [inline packages/governance/src/electorateTools.js:94]: This one is [for the use of the contractGoveror. ](https://github.com/Agoric/agoric-sdk/blob/ffe5e7ff3dd7ec1ffbbcd86e244ef167ed70c53b/packages/governance/src/contractGovernor.js#L195).  Always empty.

File: packages/governance/src/noActionElectorate.js [hunk 4]
```diff
@@ -39,8 +40,12 @@ const start = zcf => {
 
   const creatorFacet = makeHeapFarInstance('creatorFacet', ElectorateCreatorI, {
     getPoserInvitation() {
-      return zcf.makeInvitation(() => {},
-      `noActionElectorate doesn't allow posing questions`);
+      return zcf.makeInvitation(
+        () => {},
+        `noActionElectorate doesn't allow posing questions`,
+        undefined,
+        EmptyProposal,
```
Comment (Chris-Hibbert) [inline packages/governance/src/noActionElectorate.js:45]: like committee.js

File: packages/inter-protocol/src/psm/psmCharter.js [hunk 5]
```diff
@@ -4,7 +4,10 @@ import '@agoric/governance/src/exported.js';
 import { makeScalarMapStore, M, makeHeapFarInstance, fit } from '@agoric/store';
 import '@agoric/zoe/exported.js';
 import '@agoric/zoe/src/contracts/exported.js';
-import { InstanceHandleShape } from '@agoric/zoe/src/typeGuards.js';
+import {
+  EmptyProposal,
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/src/psm/psmCharter.js]: These are all correctly `Empty`

File: packages/inter-protocol/src/psm/psm.js [hunk 6]
```diff
@@ -290,6 +292,8 @@ export const start = async (zcf, privateArgs, baggage) => {
         M.splitRecord({
           give: { In: stableAmountShape },
           want: M.or({ Out: anchorAmountShape }, {}),
+          multiples: 1n,
+          exit: M.any(),
         }),
```
Comment (erights) [inline packages/inter-protocol/src/psm/psm.js:277]: This now represents all the fields of the record, not just a subset. Thus, the `M.splitRecord` should be removed and the record pattern should be the proposalShape directly.

**Comment by erights** [comment]
Closing in favor of https://github.com/Agoric/agoric-sdk/pull/7041

**Pull request closed without merging**